gluecodium/src/main/resources/templates/jni/utils/JniCppConversionUtilsImplementation.mustache

```diff
@@ -42,6 +42,12 @@ convert_string_from_jni( JNIEnv* env, jstring java_string )
     return nresult;
 }
 
+const char*
+convert_cstring_from_jni( JNIEnv* env, jstring java_string )
+{
+    return env->GetStringUTFChars( java_string, nullptr );
```

Comment:
Smells like memory leak. Who will clean this up? ReleaseStringUTFChars must be called after use.